.github/ISSUE_TEMPLATE/dev-ticket.md

```diff
@@ -7,35 +7,29 @@ assignees: ''
 
 ---
 
-<!--- Provide a general summary of the issue in the Title above -->
+**What**
+<!--- Describe the deliverable -->
 
-## Rationale
-<!--- Provide the relevant background information to udnerstand why this change is needed -->
-<!--- This is where you can provide more details thathelp the reader understand why you want to make this change -->
+**Why**
+<!--- Describe why the proposed changes are needed -->
 
-## Implementation details
-<!--- Explain the steps you will take to implement this change. If you do not have specific steps yet, write your implementation ideas or outline general approach -->
+**Context**
+<!--- Explain any relevant background -->
 
-## Priority
-<!--- How urgent is this issue, and why? -->
-<!--- Explain in writing or give a numerical value from 1 (nice-to-have) to 5 (critical) -->
+**Prerequisites/resources**
+<!--- does anything else need to happen to enable this work to be completed? -->
 
-## Scope
-<!--- Which parts of the project/codebase does this affect? -->
-<!--- What are the possibel downstream impacts or side-effects of this change?? -->
+**Statement of Work**
+<!--- what specific steps will be taken to address the aims of the ticket? -->
+<!--- Include: -->
+<!--- details of files to add/update -->
+<!--- any documentation updates required-->
+<!--- any tests that need adding/updating>
+<!--- -->
 
-## Size
-<!--- How long do you expect this to take? -->
-<!--- How many people/person-hours are required? -->
 
-## What does "done" look like?
-<!--- Explain what needs to happen in order for this to be considered complete -->
-
-## Does this require updates to documentation or other materials??
-<!--- Are there documentation, website or other media changes required after this change is implemented?. -->
-
-## Deadline
-<!--- Does this need to happen by a specific date? -->
-
-## Acceptance criteria
-<!--- What has to be true for this to get merged? What threshold of testing is acceptable? -->
+**Acceptance criteria**
```

Comment:
@jmcook1186 let's add to the template to trigger thinking in that style 

**Given** ...
**When** ...
**Then** ...